frontend/packages/data-portal/app/components/Run/ViewerPage.tsx

```diff
@@ -223,29 +295,29 @@ function ViewerPage({ run }: { run: any }) {
               <CustomDropdownSection title="Toggle annotations per deposition">
                 <CustomDropdownOption
                   selected={false}
```

Comment:
Can we make this instead be selected by an `isAnyAnnotationHidden` style function?